floss/main.py

```diff
@@ -533,15 +534,45 @@ def main(argv=None) -> int:
 
         return 0
 
-    results = ResultDocument(metadata=Metadata(file_path=sample, min_length=args.min_length), analysis=analysis)
-
     time0 = time()
     interim = time0
     sample_size = os.path.getsize(sample)
 
     static_strings = get_static_strings(sample, args.min_length)
 
-    language = identify_language(sample=sample, static_strings=static_strings)
+    language, language_version = identify_language(sample=sample, static_strings=static_strings)
+
+    if language == Language.GO:
+        enhancedStaticStrings = extract_go_strings(sample=sample, min_length=args.min_length)
+        enhancedStaticStrings = list(enhancedStaticStrings)
+
+        interim = time()
+
+        analysis = Analysis(
+            enable_static_strings=True,
+            enable_stack_strings=False,
+            enable_tight_strings=False,
+            enable_decoded_strings=False,
+        )
+
+        results = ResultDocument(metadata=Metadata(file_path=sample, min_length=args.min_length), analysis=analysis)
+
+        results.strings.static_strings = static_strings
+        results.strings.enhanced_static_strings = enhancedStaticStrings
+        results.metadata.language_version = language_version
+
+        results.metadata.runtime.total = get_runtime_diff(time0)
+        logger.info("finished execution after %.2f seconds", results.metadata.runtime.total)
```

Comment:
seems like after we print this out, floss continues running for a bit, does the rendering take more time now?